docs/SETUP.md

```diff
@@ -214,6 +214,7 @@ The CLI provides example values, but I'll also put the configuration I'm using (
 
 ```sh
 $ npm install -g pm2
+$ sudo pm2 logrotate -u sharafian # configure pm2 to use log rotation
```

Comment:
For me, the command was
```
sudo env PATH=$PATH:/home/michiel/.nvm/versions/node/v6.9.5/bin pm2 logrotate -u michiel
```

but it's close enough to guide people on their way :)